src/detect-engine.c

```diff
@@ -87,6 +87,7 @@
 #include "runmodes.h"
 
 #include "reputation.h"
+#include <malloc.h>
```

Comment:
this will need to be included in suricata-common.h, with a check in configure.ac to see if it is available